python/src/trezorlib/cli/ethereum.py

```diff
@@ -164,14 +167,96 @@ def _format_access_list(
     ]
 
 
+def _hex_or_file(data: str) -> bytes:
+    try:
+        path = Path(data)
+        if path.is_file():
+            return path.read_bytes()
+    except Exception:
+        pass
+
+    try:
+        if data.startswith("0x"):
+            data = data[2:]
+        return bytes.fromhex(data)
+    except Exception as e:
```

Comment:
`except ValueError` should be enough here

Reply:
changed in rebase